taskcluster/ci/build/kind.yml

```diff
@@ -60,7 +60,6 @@ job-defaults:
                     - ':{component}:testRelease'
                     - ':{component}:lintRelease'
                     - ':{component}:publish'
-                    - ':{component}:renameArtifacts'
```

Comment:
Where / How are we going to add `-PversionOverride=...` to the command? 🤔

Reply:
I'm reusing the old one from a few lines above (`nightlyVersion` instead of `versionOverride` - https://github.com/mozilla-mobile/android-components/pull/6116/files#diff-ec565f108a3a09f10c613b7dea5ee557L58 )

Reply:
See https://github.com/mozilla-mobile/android-components/pull/6116/files#diff-88b7c47e47b8ee65263b6784b86fa0a7R41